src/marks/arrow.js

```diff
@@ -91,6 +87,23 @@ export class Arrow extends Mark {
             // wings, but that’s okay since vectors are usually small.)
             const headLength = Math.min(wingScale * sw(i), lineLength / 3);
 
+            // Maybe flip the bending, if a sweep order is applied.
+            const flip =
+              this.sweep == null
+                ? 1
+                : this.sweep === "order-x"
+                ? descending(x1, x2)
+                : this.sweep === "order-y"
+                ? descending(y1, y2)
+                : descending(x1, x2) || descending(y1, y2); // "order"
```

Comment:
I see the use case for *order-x* and *order-y*, but what’s the use case for *order*? Can we remove it? I prefer to add as little functionality as possible unless there is an articulated use case.

Reply:
OK; it was more for “completeness” but I have not found a use case.